src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs

```diff
@@ -0,0 +1,34 @@
+namespace Particular.LicensingComponent.MonitoringThroughput;
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NServiceBus.Transport;
+using ServiceControl.Transports;
+using Shared;
+
+internal class MonitoringThroughputHostedService(ITransportCustomization transportCustomization, TransportSettings transportSettings, ILogger<MonitoringThroughputHostedService> logger, MonitoringService monitoringService) : IHostedService
```

Comment:
We use BackgroundService for all the other ones - should we keep it in sync?

Reply:
I looked at it, but unfortunately the signature methods of our transport don't really line up well with a single `run` method, hence the reason to do it that way.
This is the way the other are also done.